tests/test_sphere.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+import pytest
+import unittest
+from ddt import ddt, data, unpack
+
+import numpy as np
+from broni.shapes.primitives import Sphere
+
+
+@ddt
+class TestSphere(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    @data(
+        (np.array([[1, 1, 1]]), Sphere(0, 0, 0, 10),
```

Comment:
should Sphere(0,0,0,-10) work ?

Reply:
What shoudl be done, `abs(raduis)` or exception?

Reply:
I implemented an exception at construction time.